test/include.test.js

```diff
@@ -323,8 +274,8 @@ describe('include', function() {
 
           var passport = user.passports()[0];
           // eql instead of equal because mongo uses object id type
-          passport.id.should.eql(createdPassports[0].id);
-          passport.ownerId.should.eql(createdPassports[0].ownerId);
+          should.equal(passport.id, createdPassports[0].id);
+          should.equal(passport.ownerId, createdPassports[0].ownerId);
```

Comment:
Because should.equal works if one is buffer-type Uuis and the other is string form of Uuid.  id.should.eql does not.